### PR TITLE
Update to final release version v0.33.0, release notes

### PR DESCRIPTION
**Pull request by lakshmimsft**
# Description

Update release v0.33.0 with final release version and release notes

### CODE REVIEW

**Comment by lakshmimsft** [comment]
@willtsai, @Reshrahim , @jasonviviano : pls review/update Release Notes (highlights sections etc)

File: docs/release-notes/v0.33.0.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Today we're happy to announce the release of Radius v0.33.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
```
Comment (willtsai) [inline docs/release-notes/v0.33.0.md:12]:
```suggestion

### Access Kubernetes secrets using PodSpec

A new guide is now available with instructions on how to patch existing Kubernetes secrets using [PodSpec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) definitions and provide them to the environment of a container. Visit the [How-To Guide](https://docs.radapp.io/guides/author-apps/kubernetes/secrets-podspec/) in the Radius documentation to learn more. Thanks to @ superbeeny for the contribution!

### Codecov now enabled in the Radius repo
[Codecov](https://about.codecov.io/) has been added to the [Radius repo](https://github.com/radius-project/radius) and will compare test coverage reports and display results in all future pull requests.
```

File: docs/release-notes/v0.33.0.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+[Codecov](https://about.codecov.io/) has been added to the [Radius repo](https://github.com/radius-project/radius) and will compare test coverage reports and display results in all pull requests going forward.
+
+## Community Contributions
+
+- Thanks to @gpltaylor for contributing to user experience improvements with `rad group switch` CLI command and fixing the bug with root CA file path for Windows while installing Radius.
+- Thanks to @superbeeny for contributing to the how-to-guide on patching existing Kubernetes secrets using PodSpec.
+## Community Contributions
+- Thanks to @gpltaylor for contributing to user experience improvements with `rad group switch` CLI command and fixing the bug with root CA file path for Windows while installing Radius.
+- Thanks to @superbeeny for contributing to the how-to-guide on patching existing Kubernetes secrets using PodSpec.
```
Comment (willtsai) [inline docs/release-notes/v0.33.0.md:33]:
```suggestion
## Highlights

### Access Kubernetes secrets using PodSpec

A new guide is now available with instructions on how to patch existing Kubernetes secrets using [PodSpec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) definitions and provide them to the environment of a container. Visit the [How-To Guide](https://docs.radapp.io/guides/author-apps/kubernetes/secrets-podspec/) in the Radius documentation to learn more.

### Codecov now enabled in the Radius repo

[Codecov](https://about.codecov.io/) has been added to the [Radius repo](https://github.com/radius-project/radius) and will compare test coverage reports and display results in all pull requests going forward.

## Community Contributions

- Thanks to @gpltaylor for contributing to user experience improvements with `rad group switch` CLI command and fixing the bug with root CA file path for Windows while installing Radius.
- Thanks to @superbeeny for contributing to the how-to-guide on patching existing Kubernetes secrets using PodSpec.

```

**Review by willtsai — APPROVED** [review]

**Review by willdavsmith — APPROVED** [review]

**Review by rynowak — APPROVED** [review]
:shipit: !

**Pull request merged**